_components/banner.md

```diff
@@ -44,9 +44,12 @@ lead: The banner identifies your site as an official website of the United State
       </ul>
       <h4 class="usa-heading">Implementation</h4>
       <ul class="usa-content-list">
+        <li>A small number of .mil websites do not belong to an official U.S. Department of Defense organization. These sites may choose to adapt the <strong>Official websites use .mil</strong> section to use more technically correct language: “A <strong>.mil</strong> website operates under the approval authority of the U.S. Department of Defense.”</li>
```

Comment:
Replace `small number` with `some`, use `should`, and change `correct` to `accurate`.

<li>Some .mil websites do not belong to an official U.S. Department of Defense organization. These sites should adapt the <strong>Official websites use .mil</strong> section to use more technically accurate language: “A <strong>.mil</strong> website operates under the approval authority of the U.S. Department of Defense.”</li>